sigma/pipelines/panther/sdyaml_transformation.py

```diff
@@ -22,15 +22,19 @@ def apply(self, pipeline: ProcessingPipeline, rule: SigmaRule, query: Any) -> An
         }
 
         if rule.references:
```

Comment:
Would it make sense to check the length of this list to ensure it's 1?

Reply:
afaik empty list is always considered as False
```
>>> bool([])
False
```

Reply:
True, good call. I was thinking more along the lines of more than one element being present in the list, but that might not be a realistic concern.

Reply:
if there are >1 references we should use only first because sdyaml does not handle well list of references at this moment